engine/userguide/networking/network-debug.md

```diff
@@ -0,0 +1,212 @@
+---
+description: Learn to use the built-in network debugger to debug overlay networking problems
+keywords: network, troubleshooting, debug
+title: Debug overlay or swarm networking issues
+---
+
+Docker CE 17.12 and higher introduce a network debugging tool designed to help
+debug issues with overlay networks and swarm services running on Linux hosts.
+When enabled, a network diagnostic server listens on the specified port and
+provides diagnostic information. The network debugging tool should only be
+started to debug specific issues, and should not be left running all the time.
+
+Information about networks is stored in a database, which can be examined using
+the API. The database consists of one table per network.
```

Comment:
`The database consists of one table per network.` would remove this, it is actually 2 per network so far, but not really of interest

Reply:
How are you supposed to know what the table name is that you want to dump? I couldn't figure this out.

Reply:
the idea is that the majority of the people will use the client that abstracts this detail and deserialize the structures of the database.
The tables are the `endpoint_table` and the `overlay_peer_table`